_includes/adding_users.md

```diff
@@ -0,0 +1,12 @@
+<a name="adding_users" href="#" id="toplink">top</a>
```

Comment:
Should this file be in bitbucket rather than github?

Reply:
(Sorry to be commenting when I'm not tagged on the PR, but: bitbucket is behind the firewall, so that I think might be inaccessible. And MISO walkthrough users at other sites will still need to know how to get themselves added to MISO, so it would be good to keep instructions for adding users here. 

I wonder if this could be made more general, like "Adding Users: The person guiding the walkthrough will let you know which of the following methods to use to get your user added to MISO" and then the two options are behind clickable titles to show/hide the content. 

For OICR purposes, the user will already be added to MISO by the time they start the walkthrough, so the IT email level of specificity isn't necessary)